### ISSUE TITLE
`<Icon />` component

### ISSUE THREAD

**Issue by ptbrowne** [cozy/cozy-ui#125]
I have done an [Icon component](https://gitlab.cozycloud.cc/labs/cozy-maif/blob/master/src/components/Icon.jsx) for the MAIF project.

Example usage : 

```
<Icon icon='out' />
<Icon icon='home' />
<Icon icon='phone' />
```

The icon's color comes from the surrounding text and you can change it via CSS.

I see that in Drive, you [use `background-image`s to display icons](https://github.com/cozy/cozy-drive/search?utf8=%E2%9C%93&q=icon&type=). The benefit I see of this approach is that you can change the icon of a button without changing the HTML/JSX code. The downside is that it is not possible (?) to color the image from the surrounding text.

The `<Icon />` approach is very similar to the Font-Awesome way of doing things so external developers may be familiar with it, and you do not need to add CSS for your component to display an icon. This allows for less custom code, more reliance on `cozy-ui` and more coherent designs.

What do you think of this component and do you think it could be an addition to cozy-ui ?

[EDIT] 
When I say "icon", I am talking about this kind of icons : 
![image](https://cloud.githubusercontent.com/assets/465582/26619928/f971c8b2-45e0-11e7-8029-ad6c7c72b969.png)

Not this kind

![image](https://cloud.githubusercontent.com/assets/465582/26619952/0a30fb3c-45e1-11e7-9b1d-748d48bcb797.png)

**Comment by ptbrowne** [comment]
When I say "icon", I am talking about this kind of icons : 
![image](https://cloud.githubusercontent.com/assets/465582/26619928/f971c8b2-45e0-11e7-8029-ad6c7c72b969.png)

Not this kind

![image](https://cloud.githubusercontent.com/assets/465582/26619952/0a30fb3c-45e1-11e7-9b1d-748d48bcb797.png)

**Comment by y-lohse** [comment]
This is what I've been using mostly outside of cozy, and I find it generally a lot easier to use. The color changing alone is worth it imo.

**Comment by ptbrowne** [comment]
https://github.com/cozy/cozy-ui/pull/130

**Comment by GoOz** [comment]
This is a tricky subject. We chose, at some point, the CSS background solution, with embed svg for better performance. But we didn't have all those apps we have now, we didn't have the issues we have now.
Now may be a good time to reconsider the choice we made then. I'm strongly in favor for inline SVGs, as the browser support is good enough and the pros are so much appropriated for big webapps, but not without thinking it properly. A misuse of SVG sprite could cost a lot.
